packages/react-notifications/src/index.ts

```diff
@@ -1 +1,11 @@
 export * from './components/InAppMessaging';
+export {
```

Comment:
Let's add a comment here:
```suggestion
// re-export in-app types from '@aws-amplify/ui-react-core-notifications'
export {
```

and move the re-export in to  _./components/InAppMessaging_